values.yaml

```diff
@@ -375,6 +375,14 @@ concourse:
       ##
       database: atc
 
+      ## Postgresl service monitor (optional, default: true)
+      ##
+      serviceMonitor:
+        enabled: true
+        interval: 30s
+        labels: {}
+        relabelings:
```

Comment:
I think these options are related to postgres-operator only. Can you move it under `operator`?

Like this:

```
concourse.web.postgres.serviceMonitor -> concourse.web.postgres.operator.serviceMonitor
```

Reply:
done